### PR TITLE
@context resolution respects relative URLs.

### PR DESCRIPTION
**Pull request by ericprud**
c.f. w3c/json-ld-api#228

### CODE REVIEW

**Comment by gkellogg** [comment]
LGTM.

File: lib/documentLoaders/node.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ module.exports = ({
     const statusText = http.STATUS_CODES[res.statusCode];
     if(res.statusCode >= 400) {
       throw new JsonLdError(
-        'URL could not be dereferenced: ' + statusText,
+        'URL ' + url + ' could not be dereferenced: ' + statusText,
```
Comment (dlongley) [inline lib/documentLoaders/node.js]:
```suggestion
        `URL "${url}" could not be dereferenced: ${statusText}`,
```

File: lib/documentLoaders/node.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ module.exports = ({
     const statusText = http.STATUS_CODES[res.statusCode];
     if(res.statusCode >= 400) {
       throw new JsonLdError(
-        'URL could not be dereferenced: ' + statusText,
+        `URL "${url}" could not be dereferenced: ${statusText}`,
```
Comment (davidlehn) [inline lib/documentLoaders/node.js]: This is a change from our usual style.  URL is already a param in the details so could be omitted in the message text.  Could also move the statusText to details.

Comment (davidlehn) [reply lib/documentLoaders/node.js]: I'll merge PR, but may change this text later.

**Pull request merged**